soundbay/models.py

```diff
@@ -29,6 +29,22 @@ def _choose_block_class(block):
         module_name = '.'.join(block.split('.')[:-1])
         return getattr(importlib.import_module(module_name), class_name)
 
+    def freeze_layers(self, freeze_layers_for_finetune=False):
+        """
+        Freeze all layers except the classifier and last layer block from training, as a condition for finetune
+        Input:
+            freeze_layers_for_finetune: True if objective is to finetune the model (default == False)
+        """
+        if freeze_layers_for_finetune:
```

Comment:
looks great to me, the only thing I would change is removing the redundant `freeze_layers_for_finetune` flag
I mean, if the user calls model.freeze_layers() he probably want's to freeze layers, adding extra bool flag might be cumbersome at the happy flow or potential bug in the sad (and realistic..) flow.
So no args are needed inside `freeze_layers` imo

Reply:
Made the changes as requested